_api-reference/cluster-health.md

```diff
@@ -104,5 +104,113 @@ The following table lists all response fields.
 
 ## Required permissions
 
-If you use the security plugin, make sure you have the appropriate permissions:
-`cluster:monitor/health`.
\ No newline at end of file
+If you use the Security plugin, make sure you have the appropriate permissions:
+`cluster:monitor/health`.
+
+## Returning cluster health by awareness attribute
+Introduced 2.5
+{: .label .label-purple }
+
+We are introducing the ability to perform a cluster health check on specified awareness attributes of the cluster as an experimental feature. For updates on the progress of the feature, or if you want leave feedback that could help improve it, refer to the [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/5851).
```

Comment:
We will not mark this experimental now so we can remove this

Reply:
Done